modules/custom/az_migration/src/Plugin/migrate/process/ParagraphsMapFieldPropertyValue.php

```diff
@@ -24,6 +24,9 @@ public function transform($value, MigrateExecutableInterface $migrate_executable
     $value_key = $this->configuration['value'];
     // Get the field values.
     if ($key === 'body_format') {
+      $value[$key] = $this->configuration['default_value'];
+    }
+    elseif ($key === 'options') {
       $value[$key] = $value_key;
     }
```

Comment:
```suggestion
    if (isset($this->configuration['default_value'])) {
      $value[$key] = $this->configuration['default_value'];
    }
```

Reply:
I think it shouldn't be needed to check for specific key names, just if a default value exists the plugin is trying to set for its key.